README.md

```diff
@@ -88,31 +88,42 @@ At this point you should make a backup of the `.charon/validator_keys` folder as
 
 If taking part in the official Athena testnet, one cluster member will have to submit the `cluster-lock` and `deposit-data` files to the Obol Team, setting the stage for activation.
 
-## Step 4. Start the Distributed Validator Cluster
+## Step 4. Start the Distributed Validator Node
 
-With the DKG ceremony over, the last phase before activation is to prepare your node for validating over the long term. This repo is configured to sync an execution layer client (`geth`) and a consensus layer client (`lighthouse`).
+With the DKG ceremony over, the last phase before activation is to prepare your node for validating over the long term.
 
-**NOTE**: Update the `$BEACON_NODE_ENDPOINTS` in the `charon-config.yaml` with your beacon node URL.
+**NOTE:**
+- Update the `$BEACON_NODE_ENDPOINTS` in the `charon-config.yaml` with list of beacon nodes endpoints.
+- Update `$NAMESPACE` in all yaml files in the `manifests` directory (`manifests/*.yaml`).
 
+### Create namespace
+Choose a unique namespace name to avoid conflicts with the existing namespaces.
 ```
-# Create charon namespace
-kubectl create namespace charon
-
-# Populate .charon folder artefacts as kubernetes secrets
+NAMESPACE=<cluster_name>
+kubectl create namespace $NAMESPACE
+```
+### Create k8s secrets
+Populate the cluster config and validators keys as k8s secrets.
+```
+% cat deploy-node.sh 
 files=""
 for secret in ./.charon/validator_keys/*; do
     files="$files --from-file=./.charon/validator_keys/$(basename $secret)"
 done
-kubectl -n charon create secret generic validator-keys $files
-kubectl -n charon create secret generic charon-enr-private-key --from-file=charon-enr-private-key=./.charon/charon-enr-private-key
-kubectl -n charon create secret generic cluster-lock --from-file=cluster-lock.json=./.charon/cluster-lock.json
+kubectl -n $NAMESPACE create secret generic validator-keys $files
+kubectl -n $NAMESPACE create secret generic charon-enr-private-key --from-file=charon-enr-private-key=./.charon/charon-enr-private-key
+kubectl -n $NAMESPACE create secret generic cluster-lock --from-file=cluster-lock.json=./.charon/cluster-lock.json
+```
```

Comment:
This can put into a script file `deploy-node.sh` and simply be executed as `./deploy-node.sh`:
```
NAMESPACE=charon
kubectl create namespace $NAMESPACE

# Populate the cluster config and validators keys as k8s secrets.

files=""
for secret in ./.charon/validator_keys/*; do
    files="$files --from-file=./.charon/validator_keys/$(basename $secret)"
done

kubectl -n $NAMESPACE create secret generic validator-keys $files
kubectl -n $NAMESPACE create secret generic charon-enr-private-key --from-file=charon-enr-private-key=./.charon/charon-enr-private-key
kubectl -n $NAMESPACE create secret generic cluster-lock --from-file=cluster-lock.json=./.charon/cluster-lock.json
```